Added settings page

steel101 opened the pull request:
i have made a very simple setings page to launch ap with all the options

steel101 committed:
Create settings.py

added settings page

steel101 committed:
Update settings.py

plonibarploni commented:
I noticed you've hard-coded the network interfaces `"wlp1s0","enp2s0","lo","anbox0"`. Is that necessary, or is it possible to retrieve them dynamically from the current system?

steel101 commented:
i need some help doing that i cant figure it out

plonibarploni commented:
[This](https://stackoverflow.com/questions/3837069/how-to-get-network-interface-card-names-in-python) might help.

steel101 commented:
shur does fixing it now

steel101 committed:
Update settings.py

need help with client isolate the setting is there with all options but it wont start the hotspot with  isolate clients on but works with client option on but not with no clients need some help plz with any help

steel101 commented:
need help with client isolate the setting is there with all options but it wont start the hotspot with  isolate clients on but works with client option on but not with no clients need some help plz with any help

steel101 commented:
i understand that but i need help with making var1 var2 etc on the fly instid of hard coding them any help would be nice and i need some help with client isolation the commalnd is not running when i choose the option in settings could you help
